ArduPlane/ArduPlane.cpp

```diff
@@ -690,4 +690,14 @@ bool Plane::get_target_location(Location& target_loc)
     return false;
 }
 
+// correct AHRS pitch for TRIM_PITCH_CD in non-VTOL modes
+float Plane::get_osd_pitch_rad() const
+{
+   float pitch = ahrs.pitch;
+   if (!quadplane.in_vtol_mode()) {
```

Comment:
```suggestion
   if (!quadplane.show_vtol_view()) {
```

Reply:
The trim pitch CD is added into attitude based on in fixed wing or not, what is the difference with this?

Reply:
This is only different from `in_vtol_mode` for tailsitters, it delays the swap in attitude view to the swap of the controllers at your tailsit angle.

Reply:
reworked